INSTALL.md

```diff
@@ -74,29 +67,41 @@ that `configure.sh` is simply a wrapper around CMake that provides an easy way
 of turning on and off options. To see which options are supported, simply run
 `configure.sh -h`.
 
-    pushd $ASTROBEE_WS
+    pushd $ISAAC_WS
     ./src/scripts/configure.sh -l
     source ~/.bashrc
     popd
 
-The configure script modifies your ``.bashrc`` to source ``setup.bash`` for 
-the current ROS distribution and to set CMAKE_PREFIX_PATH. It is suggested
-to examine it and see if all changes were made correctly.
+If you run a Zsh session, then
```

Comment:
I don't use Zsh sessions so this might not apply, but we do have some scripts that add paths to the .bashrc since that is what is it used by default in ubuntu (I don't even have a ~/.zshrc file). Encouraging people to use Zsh might have unintended consequences..